Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -22,12 +22,12 @@ public ContactPolicies(IContactPoliciesService contactPoliciesService)
         [HttpGet("/accounts/{accountName}/settings")]
         public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
         {
-            // TODO: replace this validation to get UserId by accountName
-            var contactPoliciesSettings = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
-
-            if (contactPoliciesSettings == null)
+            var userFounded = await _contactPoliciesService.GetIdUserByAccountName(accountName);
+            if (userFounded == null)
                 return NotFound($"Account {accountName} does not exist.");
 
+            var contactPoliciesSettings = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
```

Comment:
In my opinion, after gathering the `idUser` using `GetIdUserByAccountName`, we should always use `iduser` as identifier:

```suggestion
            var contactPoliciesSettings = await _contactPoliciesService.GetContactPoliciesSettingsAsync(idUser);
```

Reply:
We have talked with @ms-darianbenito and @jkortega9008 and we decided to do this in another PR.